cmd/service-controller/policy_controller.go

```diff
@@ -24,13 +25,30 @@ import (
 	"github.com/skupperproject/skupper/pkg/event"
 )
 
+var (
+	staticPolicyWatchers []*client.ClusterPolicyValidator
+	policyWatcherLock    sync.Mutex
+)
+
+// AddStaticPolicyWatcher all watchers must be defined before
+// the PolicyController is started
+func AddStaticPolicyWatcher(pv *client.ClusterPolicyValidator) {
+	policyWatcherLock.Lock()
+	defer policyWatcherLock.Unlock()
+	if staticPolicyWatchers == nil {
+		staticPolicyWatchers = []*client.ClusterPolicyValidator{}
```

Comment:
I don't believe we need to explicitly create the list here? If there are any elements to be added, append will create if necessary. The only place it seems to be used is in a range operation, which again will be ok on a nil slice.

Reply:
True stuff.